config.yml

```diff
@@ -625,92 +625,101 @@ benchmarking:
-      #     - n_jobs
-
-      - title: HistGradientBoostingClassifier
-        base_lib_name: sklearn_HistGradientBoostingClassifier_best
-        other_lib_name: onnx_sklearn_HistGradientBoostingClassifier_best
-
-hpo_reporting:
-  estimators:
-    - name: sklearn_HistGradientBoostingClassifier
-      lib: sklearn
-      legend: scikit-learn
-      color: blue
-
-    - name: lightgbm
-      lib: lightgbm
-      color: red
-
-    - name: xgboost
-      lib: xgboost
-      color: green
-
-    - name: catboost_lossguide
-      lib: catboost
-      legend: catboost lossguide
-      color: purple
-
-    - name: catboost_symmetric
-      lib: catboost
-      legend: catboost symmetric
-      color: orange
+reporting:
+  index_notebook: index
+  reports:
+    - strategy: hp_match
+      notebook: scikit_learn_intelex_vs_scikit_learn
+      other_library: sklearnex
+      log_scale: True
+      parameters:
+        n_cols: 2
+      estimators:
+        - title: KNeighborsClassifier (brute force)
+          base_lib_name: sklearn_KNeighborsClassifier_brute_force
+          other_lib_name: sklearnex_KNeighborsClassifier_brute_force
+          split_bars_by_parameters:
+            - n_jobs
+
+        - title: KNeighborsClassifier (KD tree)
+          base_lib_name: sklearn_KNeighborsClassifier_kd_tree
+          other_lib_name: sklearnex_KNeighborsClassifier_kd_tree
+          split_bars_by_parameters:
+            - n_jobs
+
+        - title: KMeans (tall)
+          base_lib_name: sklearn_KMeans_tall
+          other_lib_name: sklearnex_KMeans_tall
+
+        - title: KMeans (short)
+          base_lib_name: sklearn_KMeans_short
+          other_lib_name: sklearnex_KMeans_short
+
+        - title: Logistic Regression
+          base_lib_name: sklearn_LogisticRegression
+          other_lib_name: sklearnex_LogisticRegression
+
+        - title: LinearRegression
+          base_lib_name: sklearn_LinearRegression
+          other_lib_name: sklearnex_LinearRegression
+
+        - title: Ridge
+          base_lib_name: sklearn_Ridge
+          other_lib_name: sklearnex_Ridge
+
+        - title: TSNE
+          base_lib_name: sklearn_TSNE
+          other_lib_name: sklearnex_TSNE
+
+        - title: PCA
+          base_lib_name: sklearn_PCA
+          other_lib_name: sklearnex_PCA
+
+    - strategy: hp_match
+      notebook: onnx_vs_scikit_learn
+      log_scale: True
+      other_library: onnx
+      parameters:
+        n_cols: 2
+      estimators:
+        - title: KNeighborsClassifier (brute force)
+          base_lib_name: sklearn_KNeighborsClassifier_brute_force
+          other_lib_name: onnx_sklearn_KNeighborsClassifier_brute_force
+          split_bars_by_parameters:
+            - n_jobs
+
+        # - title: KNeighborsClassifier (KD tree)
+        #   base_lib_name: sklearn_KNeighborsClassifier_kd_tree
+        #   other_lib_name: onnx_sklearn_KNeighborsClassifier_kd_tree
+        #   split_bars_by_parameters:
+        #     - n_jobs
```

Comment:
Could you please add an inline comment in the config file to explain why this case is commented out?